index.html

```diff
@@ -189,3 +189,41 @@ <h4><a href="{{'/odata-services/' | prepend: site.baseurl | prepend: site.url}}"
   </div>
   </div>
 </div>
+
+<script>
+  $(function(){
+    var myCarousel = $("#myCarousel");
+
+    $('#myCarousel.slide').carousel({
+        interval: 5000,
+        pause: "hover"
+    });
+    
+    $('input').focus(function(){
+       myCarousel.carousel('pause');
+    }).blur(function() {
+      myCarousel.carousel('cycle');
+    });
+
+    var carousel = document.getElementById('myCarousel');
+
+    var liveRegion = document.createElement('span');
+    liveRegion.setAttribute('aria-live', 'polite');
+    liveRegion.setAttribute('aria-atomic', 'true');
+    liveRegion.setAttribute('class', 'live-region sr-only');
+    carousel.appendChild(liveRegion);
+
+    var slideCaptions =[];
+
+    $.each($('#myCarousel > div.carousel-inner > div > div > h1') ,function(index, item){
```

Comment:
```suggestion
    var headers = $(myCarousel).children('div.carousel-inner > div > div > h1');
    $.each(headers ,function(index, item){
```
Non-blocking: This could improve the selector to get children of the current item only. Since this is ran once during startup will not block.